### PR TITLE
ASC update article

### PR DESCRIPTION
**Pull request by yutanglin16**

### CODE REVIEW

**Comment by JasonFreeberg** [comment]
## [Preview link](https://antares-blog-staging-pr-202.azurewebsites.net) 
- Your changes have been deployed to the preview site. The preview site will update as you add more commits to this branch. 
- The preview site shows any future-dated articles. Don't worry, if you are publishing a future-dated article, it will not show on the production site until the file's specified date.
- The preview link is shareable, but will be deleted when the pull request is merged or closed.
> *This is an automated message.*

File: _posts/2021-03-17-2020-asc-update-and-faq.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+| Renewed Certificate Validity | |  Time Before Expiration for Renewal | | Total Time Validity for Renewed Certificate | Meets New 13 Month Validity Policy? | Result |
+| --- | :--: | --- | :---: | --- | --- | --- |
+| 1 year (12 months) of validity | + | 2 months before expiration | = | 14 months | No | Renewed certificate expiration will appear ~1 month shorter (refer to [getting a certificate with "shorter" expiration]()) |
+| 1 year (12 months) of validity | + | 1 month before expiration | = | 13 months | Yes | Renewed certificate expiration will be the same (might expect ~1 day off) |
+
+You can still manually renew your certificate up to 60 days before expiration. We are keeping this option open in case you need to renew your certificate earlier, but be aware that the 13-month validity policy will still apply. Refer to [getting a certificate with "shorter" expiration]() section of the article for more information.
```
Comment (JasonFreeberg) [inline _posts/2021-03-17-2020-asc-update-and-faq.md:58]: Missing link for "getting a certificate with "shorter" expiration"

File: _posts/2021-03-17-2020-asc-update-and-faq.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Getting a certificate with "shorter" expiration
+
+If your certificate is affected by the 13-month validity policy change and have been issued a certificate with a "shorter" expiration, we will not forego the remaining time on the certificate.
```
Comment (JasonFreeberg) [inline _posts/2021-03-17-2020-asc-update-and-faq.md:62]: Can we try rephrasing this? I don't understand what is meant by "we will not forego the remaining time on the certificate"

File: _posts/2021-03-17-2020-asc-update-and-faq.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+There will be a "subscription" policy put in place for your certificate should your renewed certificate have an "shorter" expiration date. Meaning, we will keep a record of the "original" expiration of your certificate and will issue another certificate 30 days before the new "shorter" expiration that will expire until your "orginal" expiration date. If you have auto-renew on, we will just extend the "original" expiration by another year. Refer to the chart below:
```
Comment (JasonFreeberg) [inline _posts/2021-03-17-2020-asc-update-and-faq.md:64]: Last sentence mentions a chart but there isn't one

**Review by JasonFreeberg — CHANGES_REQUESTED** [review]

**Review by mster1222 — COMMENTED** [review]
work

**Pull request closed without merging**